### PR TITLE
chore: bump alpine to 3.19, bump php to 8.3, config features, fix instances.json + readme.md

### LINKED ISSUE #93: PHP 8.3 Release
See:
https://www.php.net/releases/8.3/en.php

This shouldn't affect any of our code right now, nor the documentation, right?
I can test in a Debian virtual machine, it's packaged in experimental:
https://packages.debian.org/experimental/php8.3

### PR DESCRIPTION
**Pull request by codedipper**
Closes #93 

TODO:

- [x] Bump Alpine to 3.19
- [x] Bump PHP to 8.3
- [x] Confirm it works
- [x] Update configuration features for Docker
- [x] Add 3rd Ahwx instance to README
- [x] Reorder instances.json
- [x] Remove broken instances

### CODE REVIEW

**Comment by codedipper** [comment]
Broken instances:
https://search2.ahwx.org/ - Clearnet only, SSL_ERROR_UNRECOGNIZED_NAME_ALERT
https://search3.ahwx.org/ - Clearnet only, SSL_ERROR_UNRECOGNIZED_NAME_ALERT
https://librex.baczek.me/ - Taken down voluntarily
https://librex.ix.tc/ - No DNS record
https://librex.yogeshlamichhane.com.np/ - Captcha
https://librey.danyaal.xyz/ - Clearnet, Tor & I2P, SSL_ERROR_BAD_CERT_DOMAIN, Unable to connect, Host is down
https://search.ratakor.com/ - Keeps timing out

search2.ahwx.org and search3.ahwx.org appear to contain the same A record.

**Comment by codedipper** [comment]
Nighty night my little European, see you when you merge this in the morning :)

**Comment by Ahwxorg** [comment]
> Broken instances:
> https://search2.ahwx.org/ - Clearnet only, SSL_ERROR_UNRECOGNIZED_NAME_ALERT
> https://search3.ahwx.org/ - Clearnet only, SSL_ERROR_UNRECOGNIZED_NAME_ALERT

Both should not be clearnet only, and both should be up. I will yet again change shit.

It's the same DNS A record because of proxy reasons.

**Comment by davidovski** [comment]
https://librex.ix.tc/ is supposed to be https://librey.ix.tc/,  not sure how i managed to put the wrong domain when the commit message adding it had the right domain.

**Comment by Ahwxorg** [comment]
Please ping me whenever this is ready to merge.

File: config.php.example
Comment (davidovski) [inline config.php.example:1]: This is quite an important change. Maybe its worth putting out a message to LibreY hosters telling them to switch to "auto" to get more consistent results.

**Review by davidovski — APPROVED** [review]

**Comment by codedipper** [comment]
@Ahwxorg whenever your ready, everything works. :smile: 
Please take care of yourself though, we love you and we're all here for you if you have to talk.

**Pull request merged**